base/special/bessel.jl

```diff
@@ -40,95 +39,81 @@ let
     end
 end
 
+
 """
-    airy(k,x)
+    airyai(x)
 
-The `k`th derivative of the Airy function ``\\operatorname{Ai}(x)``.
+Airy function of the first kind ``\\operatorname{Ai}(x)``.
 """
-function airy(k::Integer, z::Complex128)
-    id = Int32(k==1 || k==3)
-    if k == 0 || k == 1
-        return _airy(z, id, Int32(1))
-    elseif k == 2 || k == 3
-        return _biry(z, id, Int32(1))
-    else
-        throw(ArgumentError("k must be between 0 and 3"))
-    end
-end
+function airyai end
+airyai(z::Complex128) = _airy(z, Int32(0), Int32(1))
 
 """
-    airyprime(x)
+    airyaiprime(x)
 
-Airy function derivative ``\\operatorname{Ai}'(x)``.
+Derivative of the Airy function of the first kind ``\\operatorname{Ai}'(x)``.
 """
-airyprime(z) = airy(1,z)
-@vectorize_1arg Number airyprime
```

Comment:
Actually the vectorized forms aren't deprecated; I just moved the `@vectorize_1arg` calls to the `@eval` statement further down.